assets/stylesheets/responsive/custom.scss

```diff
@@ -1229,3 +1253,37 @@ dt {
 dd {
   margin-left: 0;
 }
+
+// Pagination
+div.pagination {
+  // using div.pagination here to override specificity
+  padding-top: 1em;
+  margin-bottom: 1em;
+}
+
+div.pagination span.current,
+div.pagination a,
+div.pagination a:visited,
+.pagination .previous_page,
+.pagination .next_page,
+.pagination .current {
+  @extend .button-secondary;
+  margin: 0 0.2em;
```

Comment:
There is - it uses the .button-secondary one, probably hard to spot, due to the bugs we talked about earlier that are solved in #216